babel.config.js

```diff
@@ -12,13 +12,17 @@ module.exports = (api) => {
         '@babel/preset-env',
         {
           targets: {
-            // TODO: can we get this to read from package.json somehow..?
-            node: '12',
+            node: 'current',
```

Comment:
This is subjective. We should hard code this otherwise it's inherited from whatever workflow node version is running which can make updating the `setup-node` flow breaking.

Reply:
I hardcoded node14.